beacon-chain/powchain/service.go

```diff
@@ -571,3 +510,71 @@ func safelyHandlePanic() {
 		}).Error("Panicked when handling data from ETH 1.0 Chain! Recovering...")
 	}
 }
+
+func (w *Web3Service) handleDelayTicker() {
+	defer safelyHandlePanic()
+	if w.lastRequestedBlock.Cmp(w.blockHeight) == 0 {
```

Comment:
Why only if they are the same? I dont understand the logic of this method without additional context. Please add comments or refactor for clarity

Reply:
No idea, didn't add this stuff unfortunately

Reply:
@nisdas what is the purpose of this statement?

Blame shows you added it here: https://github.com/prysmaticlabs/prysm/pull/1605/files#diff-119011374e72469b7f72390c56bc95fbR389